packages/jest-environment-puppeteer/README.md

```diff
@@ -60,6 +60,20 @@ it('should fill an input', async () => {
 })
 ```
 
+### `global.context`
+
+Give access to a [Browser context](https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#class-browsercontext) that is instanciated when the browser is launched.
+
+It is possible to set the browser context inside the config `jest-puppeteer-config.js` in the root of this project. The context will always be exposed via the global `context` object the same way `page` and `browser` are exposed. 
+
+Accepted values: 
```

Comment:
This is not the right place for it, see below ` ### jest-puppeteer.config.js`

Reply:
Will fix soon ! Thanks for the feedback :) !